### PR TITLE
minor cleanups

### PR DESCRIPTION
**Pull request by urso**
Some minor code cleanups to make linters a little more happy (thanks goes to `gofmt -s` and `gorename`).

Includes:
- renames in libbeat to use camel case
- fix some spelling errors in comments
- modernize/simplify some code via `gofmt -s`
- check all ineffective assignments not being potential bugs (thus time no potential bugs have been found)
- ensure mutex-values are not accidentally copied

### CODE REVIEW

File: libbeat/common/net.go [hunk 1]
```diff
@@ -5,9 +5,9 @@ import (
 	"net"
 )
 
-// LocalIpAddrs finds the IP addresses of the hosts on which
+// LocalPpAddrs finds the IP addresses of the hosts on which
```
Comment (andrewkroh) [inline libbeat/common/net.go]: There's an extra `p` in the function name.

**Comment by tsg** [comment]
jenkins, test it

**Comment by tsg** [comment]
LGTM

File: libbeat/common/net.go (hunk 1 above)
Comment (urso) [inline libbeat/common/net.go]: thanks, fixed.

**Comment by ruflin** [comment]
jenkins, retest it

**Pull request merged**